lib/ansible/modules/network/netact/netact_cm_command.py

```diff
@@ -0,0 +1,364 @@
+    sample: '/opt/oss/bin/racclimx.sh -op Upload -opsName Uploading_testi -DN PLMN-PLMN/MRBTS-746'
+message:
+    description: The output message that the netact_cm_command module generates
+    returned: Command output message
+    type: string
+changed:
+    description: data changed
+    returned: true if data is changed
+    type: boolean
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+
+racclimx = '/opt/oss/bin/racclimx.sh'
+
+
+def run_module():
+    """
+    Main module where option are handled and command is executed
+    :return:
+    """
+    # define the available arguments/parameters that a user can pass to
+    # the module
+    module_args = dict(
+        operation=dict(type='str', required=True,
+                       aliases=['op'],
+                       choices=['Upload', 'Provision', 'Import',
+                                'Export', 'Provision_Mass_Modification']),
+        opsname=dict(type='str', required=False, aliases=['opsName']),
+        dn=dict(type='str', required=False, aliases=['DN']),
+        ws=dict(type='str', required=False, aliases=['WS']),
+        mr=dict(type='str', required=False, aliases=['MR']),
+
+        planname=dict(type='str', required=False, aliases=['planName']),
+        typeoption=dict(type='str', required=False, aliases=['type'],
+                        choices=['plan', 'actual', 'reference', 'template', 'siteTemplate']),
+        fileformat=dict(type='str', required=False, choices=['CSV', 'RAML2', 'XLSX'],
+                        aliases=['fileFormat']),
+        filename=dict(type='str', required=False, aliases=['fileName']),
+
+        createBackupPlan=dict(type='str', required=False, choices=['true', 'false'],
+                              aliases=['createBackupPlan']),
+        backupPlanName=dict(type='str', required=False, aliases=['backupPlanName']),
+        inputFile=dict(type='str', required=False),
+
+        verbose=dict(type='str', required=False),
+        extra_opts=dict(type='str', required=False)
+    )
+
+    # seed the result dict in the object
+    # we primarily care about changed and state
+    # change is if this module effectively modified the target
+    # state will include any data that you want your module to pass back
+    # for consumption, for example, in a subsequent task
+    result = dict(
+        changed=False,
+        original_message='',
+        message=''
+    )
+
+    # the AnsibleModule object will be our abstraction working with Ansible
+    # this includes instantiation, a couple of common attr would be the
+    # args/params passed to the execution, as well as if the module
+    # supports check mode
+    module = AnsibleModule(
+        argument_spec=module_args,
+        supports_check_mode=True
+    )
+
+    # if the user is working with this module in only check mode we do not
+    # want to make any changes to the environment, just return the current
+    # state with no modifications
+    if module.check_mode:
```

Comment:
I am not sure if you have anything presentable in the result with `check mode` here. Here is a brief example on how `check mode` is perceived here: If check_mode is set, and say a module is configuring IP address on an interface, the module would show the output of actual task performed minus the final `commit` operation that bring the task in force.

Reply:
Setting this to:
        result['skipped'] = True
        result['msg'] = 'check mode not (yet) supported for this module'
        module.exit_json(**result)